kf_model_fhir/app.py

```diff
@@ -278,6 +299,9 @@ def _custom_validate(resource_dicts):
                 f'<resource type>-<resource id>.json. File {filename}{ext} '
                 f'should be: {expected_filename}'
```

Comment:
Shouldn't these lines to be:

```python
f'<resource type>-<resource id>.json. File {filename}{ext} '
f'should be: {expected_filename}{ext}'
```

since `expected_filename` is just a file name without an extension?